dev-packages/test-utils/src/event-proxy-server.ts

```diff
@@ -115,14 +115,28 @@ export async function startEventProxyServer(options: EventProxyServerOptions): P
             eventCallbackListeners.forEach(listener => {
               const rawSentryResponseBody = Buffer.concat(sentryResponseChunks).toString();
 
-              const data: SentryRequestCallbackData = {
-                envelope: parseEnvelope(proxyRequestBody),
-                rawProxyRequestBody: proxyRequestBody,
-                rawSentryResponseBody,
-                sentryResponseStatusCode: sentryResponse.statusCode,
-              };
-
-              listener(Buffer.from(JSON.stringify(data)).toString('base64'));
+              try {
```

Comment:
I also added this to clean up logs here a bit, replay events will always fail to parse because they are compressed and not JSON.